<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TypeMappingForm</class>
 <widget class="QWidget" name="TypeMappingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Type Mapping</string>
  </property>
  <widget class="GlyphView" name="view_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="GlyphView" name="view_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>html</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>xml</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>tex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cpp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hpp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>other</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>481</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlyphView</class>
   <extends>QGraphicsView</extends>
   <header>Mapping/GlyphView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
